x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -218,6 +213,15 @@ IDENTIFIER
     : (LETTER | '_' | '@') (LETTER | DIGIT | '_')*
     ;
 
+ESCAPED_IDENTIFIER
+    : '`' (~'`')* '`'
+    ;
+
+eventValue
+    : STRINGIDENTIFIER
```

Comment:
I don't see definition for `STRINGIDENTIFIER` (should be `STRING_IDENTIFIER`).

Reply:
Sorry for those, just a mixup during merge.